beacon-chain/rpc/eth/beacon/handlers_pool.go

```diff
@@ -168,3 +171,85 @@ func (s *Server) SubmitAttestations(w http.ResponseWriter, r *http.Request) {
 		http2.WriteError(w, failuresErr)
 	}
 }
+
+// ListVoluntaryExits retrieves voluntary exits known by the node but
+// not necessarily incorporated into any block.
+func (s *Server) ListVoluntaryExits(w http.ResponseWriter, r *http.Request) {
+	_, span := trace.StartSpan(r.Context(), "beacon.ListVoluntaryExits")
+	defer span.End()
+
+	sourceExits, err := s.VoluntaryExitsPool.PendingExits()
+	if err != nil {
+		http2.HandleError(w, "Could not get exits from the pool: "+err.Error(), http.StatusInternalServerError)
+		return
+	}
+	exits := make([]*shared.SignedVoluntaryExit, len(sourceExits))
+	for i, e := range sourceExits {
+		exits[i] = shared.SignedVoluntaryExitFromConsensus(e)
+	}
+
+	http2.WriteJson(w, &ListVoluntaryExitsResponse{Data: exits})
+}
+
+// SubmitVoluntaryExit submits SignedVoluntaryExit object to node's pool
+// and if passes validation node MUST broadcast it to network.
+func (s *Server) SubmitVoluntaryExit(w http.ResponseWriter, r *http.Request) {
+	ctx, span := trace.StartSpan(r.Context(), "beacon.SubmitVoluntaryExit")
+	defer span.End()
+
+	if r.Body == http.NoBody {
+		http2.HandleError(w, "No data submitted", http.StatusBadRequest)
+		return
+	}
+	var req shared.SignedVoluntaryExit
+	if err := json.NewDecoder(r.Body).Decode(&req); err != nil {
+		http2.HandleError(w, "Could not decode request body: "+err.Error(), http.StatusBadRequest)
+		return
+	}
+	validate := validator.New()
+	if err := validate.Struct(req); err != nil {
+		http2.HandleError(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	exit, err := req.ToConsensus()
+	if err != nil {
+		http2.HandleError(w, "Could not convert request exit to consensus exit: "+err.Error(), http.StatusBadRequest)
```

Comment:
```suggestion
		http2.HandleError(w, "Could not convert exit request to consensus exit: "+err.Error(), http.StatusBadRequest)
```

Reply:
Elsewhere I use the same sentence structure, so I would prefer keeping it like this.